src/embed.js

```diff
@@ -26,10 +26,10 @@ matchers.push(spotify);
 function toHTML({
   oEmbed = {}, open_graph = {}, twitter_card = {}, other = {},
   title: otherTitle, description: otherDescription, classname,
-}, fallbackURL) {
+}, fallbackURL, lvl2Dom) {
```

Comment:
I would call it `type` or `kind` here....it could be that in the future, several domains map to the same kind.